pkg/pipelines/bootstrap.go

```diff
@@ -311,3 +318,8 @@ func checkPipelinesFileExists(appFs afero.Fs, outputPath string, overWrite bool)
 	}
 	return nil
 }
+
+//logs Success message to the outputs
+func logBootstrapStatus(message string) {
+	log.Success(message)
```

Comment:
Using `log.Success` everywhere may not be a good idea. Usually, success messages will be shown in the end. We have to follow the conventions used in other `odo` commands

Reply:
log.info in accordance with other odo commands has been incorporated.